### PR TITLE
AWS CloudTrail Password Discovery detection

### PR DESCRIPTION
**Pull request by natezpanther**
### Background

Added AWS CloudTrail detection that looks for non-service-to-service events where Password Policy discovery is taking place.

### CODE REVIEW

**Comment by andrea-youwakim** [comment]
is there a reason why this rule is marked as enabled:false?

**Comment by andrea-youwakim** [comment]
tagging @natezpanther for ease of visibility

File: rules/aws_cloudtrail_rules/aws_cloudtrail_password_policy_discovery.yml [hunk 1]
```diff
@@ -0,0 +1,56 @@
+AnalysisType: rule
+Description: This detection looks for *AccountPasswordPolicy events in AWS CloudTrail logs. If these events occur in a short period of time from the same ARN, it could constitute Password Policy reconnaissance.
+DisplayName: AWS CloudTrail Password Policy Discovery
+Enabled: false
```
Comment (egibs) [inline rules/aws_cloudtrail_rules/aws_cloudtrail_password_policy_discovery.yml:4]:
```suggestion
Enabled: true
```

File: rules/aws_cloudtrail_rules/aws_cloudtrail_password_policy_discovery.yml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+Filename: aws_cloudtrail_password_policy_discovery.py
+Reports:
+    MITRE ATT&CK:
+        - TA0007:T1201
+Severity: Info
+Tests:
```
Comment (egibs) [inline rules/aws_cloudtrail_rules/aws_cloudtrail_password_policy_discovery.yml:10]: Are these tests representative of actual log structure?

Comment (arielkr256) [reply rules/aws_cloudtrail_rules/aws_cloudtrail_password_policy_discovery.yml:10]: Looks correct according to this, for fields referenced by the rule: https://docs.aws.amazon.com/awscloudtrail/latest/userguide/cloudtrail-log-file-examples.html#cloudtrail-log-file-examples-iam

**Review by arielkr256 — APPROVED** [review]
Enabled rule and added it to AWS pack.  Good MITRE tagging!

**Pull request merged**